Musakai.Blazor/Pages/Home.razor

```diff
@@ -42,4 +76,20 @@
         new ImageSliderContent { Src = "image2.jpg"},
         new ImageSliderContent { Src = "image3.jpg"}
     };
+
+    private ContactModel contactModel = new ContactModel(); 
+    
+    private void HandleValidSubmit()
+    { 
+        // This method will send request to the server.
```

Comment:
throws NotImplementedYet?

Reply:
Throwing.

Reply:
I see NotFiniteNumberException

Reply:
My fault, IntelliSense showed me more results and I pressed enter and didn't check that...